### PR TITLE
[PG] add checking for increasing ingress PG drop packet counters

### PR DESCRIPTION
**Pull request by ayurkiv-nvda**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 201911

#### What is the motivation for this PR?
Add ability to check PG dropped packet counters

#### How did you do it?
Use existing QoS infrastructure to check dropped packets per PG per Interface

#### How did you verify/test it?
run test: 

 `py.test qos/test_qos_sai.py::TestQosSai::testQosSaiPfcXoffLimit  --inventory "../ansible/inventory, ../ansible/veos" --host-pattern (dut)-ptf32 --module-path                ../ansible/library/ --testbed (dut)-ptf32 --testbed_file ../ansible/testbed.csv`

#### Supported testbed topology if it's a new test case?
ptf32

### CODE REVIEW

File: tests/saitests/sai_qos_tests.py [hunk 1]
```diff
@@ -672,6 +673,9 @@ def runTest(self):
         # get counter names to query
         ingress_counters, egress_counters = get_counter_names(sonic_version)
 
+        # Clear all counters for all ports
+        sai_thrift_clear_all_counters(self.client)
```
Comment (neethajohn) [inline tests/saitests/sai_qos_tests.py:677]: Not sure if this works for all vendors. Can you take the counter snapshot prior to the test run and after and compare as is done for all the testcases?

File: tests/saitests/sai_qos_tests.py [hunk 2]
```diff
@@ -781,6 +785,14 @@ def runTest(self):
             for cntr in egress_counters:
                 assert(xmit_counters[cntr] == xmit_counters_base[cntr])
 
+            # According to SONiC configuration lossless dscps are classified as follows:
+            # dscp  3 -> pg 3
+            # dscp  4 -> pg 4
+            pg_dropped_cntrs = sai_thrift_read_pg_drop_counters(self.client, port_list[src_port_id])
```
Comment (neethajohn) [inline tests/saitests/sai_qos_tests.py:791]: Please check the sonic version and execute this block. These tests also run against 201811 and these counters are not supported there.  there is a similar check in get_counter_names()

File: tests/saitests/sai_qos_tests.py (hunk 1 above)
Comment (ayurkiv-nvda) [reply tests/saitests/sai_qos_tests.py]: Done

File: tests/saitests/sai_qos_tests.py (hunk 2 above)
Comment (ayurkiv-nvda) [reply tests/saitests/sai_qos_tests.py]: Done

**Review by neethajohn — APPROVED** [review]

**Comment by liat-grozovik** [comment]
@neethajohn and @ayurkiv-nvda according to the code it should run on any branch after 201811. does this mean it should be cherry picked to 201911 branch?

**Pull request merged**